dali/test/python/operator_1/test_crop.py

```diff
@@ -264,13 +264,20 @@ def __init__(self, device, batch_size, iterator, data_shape, data_layout, num_th
         else:
             assert False
 
-        self.crop = ops.Crop(device=self.device,
-                             crop_pos_z=0.1,
-                             crop_pos_y=0.2,
-                             crop_pos_x=0.3,
-                             crop_d=D * 0.91,
-                             crop_h=H * 0.85,
-                             crop_w=W * 0.75)
+        if not fixed_window:
```

Comment:
Nitpick for readability: `if not X` in a single if/else reads better as `if X` with swapped if/else branches. Up to you.

Reply:
Done